migrate.md

```diff
@@ -0,0 +1,111 @@
+---
+
+copyright:
+  years: 2015, 2019
+lastupdated: "2019-04-12"
+
+keywords: migrate,watson,Cloud Foundry
+
+subcollection: watson
+
+---
+
+{:shortdesc: .shortdesc}
+{:new_window: target="_blank"}
+{:deprecated: .deprecated}
+{:important: .important}
+{:note: .note}
+{:tip: .tip}
+{:pre: .pre}
+{:codeblock: .codeblock}
+{:screen: .screen}
+{:javascript: .ph data-hd-programlang='javascript'}
+{:java: .ph data-hd-programlang='java'}
+{:python: .ph data-hd-programlang='python'}
+{:swift: .ph data-hd-programlang='swift'}
+
+# Migrating from Cloud Foundry
+{: #migrate}
+
+Migrate a service instance to move it from its current Cloud Foundry org and space to a resource group.
+{: shortdesc}
+
+{{site.data.keyword.cloud_notm}} moved from using Cloud Foundry to using resource groups.
+
```

Comment:
Do we need this linebreak?  It seems like these two sentences might work better as a single paragraph?